src/Dnn.PersonaBar.Library/Controllers/IPersonaBarUserSettingsController.cs

```diff
@@ -28,11 +28,11 @@ public interface IPersonaBarUserSettingsController
         /// <summary>
         /// Updates the PersonaBar user settings
         /// </summary>
-        void UpdatePersonaBarUserSettings(UserSettings settings, int userId, int portalId);
+        void UpdatePersonaBarUserSettings(PersistSettings settings, int userId, int portalId);
```

Comment:
Hm, this will be a breaking change already in 9.2.1.

Reply:
I searched in all 4 PB repos there is no project referenced to this API,